### ISSUE TITLE
Integrate measurement scripts into dev branch

### ISSUE THREAD

**Issue by prasadtalasila** [prasadtalasila/BITS-Darshini#120]
We have measurement scripts written in bash shell that measure the performance of Bro, Ntopng, tshark and Darshini (both persistent and non-persistent modes). At present, the scripts are in [measurement branch](https://github.com/prasadtalasila/BITS-Darshini/tree/measurements). These scripts need to be integrated into scripts/measurements directory of dev branch.

The idea is to make suitable changes to the scripts such that the scripts run on all the [pcap files available in the repo](https://github.com/prasadtalasila/BITS-Darshini/tree/measurements/data/packet) and produce a small table of results for comparison.  We can use the existing [P4 graph](https://github.com/prasadtalasila/BITS-Darshini/blob/measurements/data/graph.p4) for running Darshini.

The scripts present in measurement branch were written in adhoc fashion. Careful thought needs to be given to modularizing them perhaps using bash shell functions. Once the scripts are in-place, we can run them in travis for all the builds.

**Comment by raghavagrawal** [comment]
After running the scripts there are two errors which cannot be resolves. 
1. Error when running the script(tool-run.sh) as root

.
2. *** Error in `ntopng': free(): invalid pointer: 0x00000000008805c0 *** when running ntopng.
Running script without ntopng and sudo works fine.

![screenshot from 2017-10-23 16-28-02](https://user-images.githubusercontent.com/26343670/31887678-dbe61516-b816-11e7-8c6c-501cb29dedae.png)

![screenshot from 2017-10-23 16-35-25](https://user-images.githubusercontent.com/26343670/31887691-f110e894-b816-11e7-8cd4-da9ff14c951b.png)

**Comment by prasadtalasila** [comment]
When you install ntopng, it installs an auto start system service. You shut down that service, using the command    
```bash
> sudo su
> service ntopng stop
> exit
```
You can try the script again and check the running status of ntopng part. I am sure there is a way in Ubuntu/Linux to stop a system service from auto starting on reboot. If you can stop ntopng from auto starting, then the problem would permanently go away.

Try running the script as a normal user and look for errors. It would be better to run the script as a normal user.

**Comment by raghavagrawal** [comment]
ntopng can be stopped from auto starting on reboot using the following command. 
 `sudo update-rc.d ntopng disable`
But still, the problem is there. Below is the screenshot of the performance.log file.
![screenshot from 2017-10-25 15-59-16](https://user-images.githubusercontent.com/26343670/31994270-a4898f5c-b99d-11e7-9f79-a81e5264a4a5.png)

**Comment by prasadtalasila** [comment]
@raghavagrawal Just confirming! You stopped the current running instance of ntopng using the command.
```bash
> sudo service ntopng stop
```
You can check the running status of ntopng using the command
```bash
> sudo service ntopng status
```

**Comment by raghavagrawal** [comment]
@prasadtalasila I did that sir.

**Comment by prasadtalasila** [comment]
@raghavagrawal There seems to be a problem with the recent build of ntopng. It is running and completing the analysis; but the application is having a hard time gracefully shutting down. 

Don't worry about the issue. There is nothing we can do about it. The future builds of ntopng should resolve the issue. However, I do encourage you to raise the issue on [ntopng](https://github.com/ntop/ntopng/issues) page. Do remember to mention the complete details so that Prof. Luca Deri, the core developer of ntopng, has a chance to rectify the problem.

You can go ahead with writing the rest of the scripts.

**Comment by prasadtalasila** [comment]
solved using PR #132.
